source/proximity_list/proximity_list.h

```diff
@@ -67,7 +67,7 @@ struct ProximityList
     };
 
     /// Contient toutes les coordonnées de manière à trouver rapidement
-    std::vector<Item> items;
+    std::vector<Item> items{};
```

Comment:
hmmm, nope, i did this because the compiler complained about no init member, which is weird, too...